lib/utils/ast-utils.js

```diff
@@ -156,6 +156,72 @@ function createIdentifierOrLiteral(j, val) {
 	return j.literal(literalVal);
 }
 
+/**
+ *
+ * Adds or updates the value of a key within a root
+ * webpack configuration property that's of type Object.
+ *
+ * @param {any} j — jscodeshift API
+ * @param {Node} rootNode - node of root webpack configuration
+ * @param {String} configProperty - key of an Object webpack configuration property
+ * @param {String} key - key within the configuration object to update
+ * @param {Object} value - the value to set for the key
+ */
+
+function addOrUpdateConfigObject(j, rootNode, configProperty, key, value) {
```

Comment:
Wow... this can actually increase the code quality of #401. Good to have!

I don't know if i copy this directly to there, or if i wait and rebase later. What you guys think it is better?

Reply:
You could copy the file for now, and use it locally and adapt your code accordingly. Once this PR is merged, you can rebase and update your PR once for all.
This utility is widely useful, so I'd wait the merge

Reply:
@bitpshr could you create a test case for this?

Reply:
I don't think it's needed because it's indirectly tested by the snapshot testing